tests/conftest.py

```diff
@@ -0,0 +1,86 @@
+import pytest
+import json
+import apypie
+import requests
+
+from collections import namedtuple
+
+User = namedtuple('User', ['username', 'password', 'name'])
+
+
+def pytest_generate_tests(metafunc):
+    with open(metafunc.config.option.variables[0]) as f:
+        variables = json.load(f)
+    assert variables
```

Comment:
I think this accesses the raw option. Note that pytest-variables also accepts other file formats, such as YAML. It also supports layering which you're breaking here. That's why I used `metafunc.config._variables` to get the user object. Not great either, but you don't break functionality.

Reply:
Sorry, I do not understand what raw option or breaking layering means here.

Reply:
I think Ewoud means the fact that `pytest-variables` can (and does) load variables itself. And it can load from multiple files, merging the result: https://github.com/pytest-dev/pytest-variables#specifying-variables

So I think you should be able to use

```python
variables = metafunc.config._variables  # pylint: disable=protected-access
```